introduction/examples/thompson/friends.py

```diff
@@ -0,0 +1,27 @@
+# zadanie 1
+friends = {"John": "polite", "Andrew": "kind", "Suzy": "wise", "Claire": "joyful", "Brian": "kind"}
+
+# zadanie 2
+friends_names = list(friends.keys())
+special_trades_list = list(set(friends.values()))
+
+# zadanie 3
+special_trades = [e for e in friends.values() if list(friends.values()).count(e) < 2]
+
+# zadanie 4
+friends_trades = {}
+
+for person in friends:
+    if person.startswith("A"):
+        friends_trades.update({person: [friends[person], "awesome"]})
```

Comment:
Do słownika można dodawać nowe klucze również tak `friends_trades[key] = value`
Jest to preferowane rozwiązanie (nad `update`) jeżeli dodajemy po jednym kluczu na raz.